src/main.rs

```diff
@@ -87,3 +79,49 @@ fn parse_args() -> Result<LndCfg, ArgsError> {
 
     Ok(LndCfg::new(address, cert_file, macaroon_file))
 }
+
+/// Sets the onion messaging feature bit (described in this PR: 
+/// https://github.com/lightning/bolts/pull/759/), to signal that we support
+/// onion messaging. This needs to be done every time we start up, because LND
+/// does not currently persist the custom feature bits that are set via the RPC.
+async fn set_feature_bit(mut client: Client) {
+    let update = tonic_lnd::peersrpc::UpdateFeatureAction {
+        action: 0,
```

Comment:
Good call looks like there is an updateaction enum

Reply:
Updated to use the enum. It seems like it won't take the enum directly and needs it converted to i32, which is maybe a bit confusing looking, but still seems more readable to me because at least it says "add" somewhere in there?